src/main/java/io/cdap/plugin/salesforce/parser/SalesforceQueryVisitor.java

```diff
@@ -158,6 +158,30 @@ public SObjectDescriptor.FieldDescriptor visitFieldName(SOQLParser.FieldNameCont
       return new SObjectDescriptor.FieldDescriptor(nameParts, alias, SalesforceFunctionType.NONE);
     }
 
+    @Override
+    public SObjectDescriptor.FieldDescriptor visitSelectKeyField(SOQLParser.SelectKeyFieldContext ctx) {
+      List<String> nameParts = ctx.selectKey().stream()
+        .map(RuleContext::getText)
+        .collect(Collectors.toList());
+
+      if (nameParts.isEmpty()) {
+        throw new SOQLParsingException("Invalid column name: " + ctx.getText());
+      }
+
+      if (nameParts.size() > 1) {
+        // if field contains alias, remove it to get actual field name
+        // Example:
+        // select Opportunity.Name from Opportunity -> Name
```

Comment:
Is the query case sensitive?
e.g: is `select opportunity.name from opportunity` (all lowercase) a valid query?

Reply:
No, the query is not case-sensitive.

However the SOQL query: `select  opportunity.name from opportunity` will be transformed by the code to `select name from opportunity` to make it valid.

Reply:
I see. Can we add a unit test for a case insensitive query with alias like `select o.name from opportunity o`?

Reply:
added cbbb7f710fd0d20e036245323bf178ebfc4d5279